sdks/python/apache_beam/transforms/environments.py

```diff
@@ -234,6 +257,35 @@ def from_options(cls, options):
     raise NotImplementedError
 
 
+@Environment.register_urn(common_urns.environments.DEFAULT.urn, None)
+class DefaultEnvironment(Environment):
+  """Used as a stub when context is missing a default environment."""
+  def __init__(
```

Comment:
You can simply omit the constructor if all it does is call super.

Reply:
Done